### PR TITLE
Implemented the components and UI design

### PR DESCRIPTION
**Pull request by lawrahkonwea**
This is the UI for the math app using React;
- Developed just the UI of the calculator following the template by Microverse.
- Used class-based components
- Once the Calculator was ready, I import it and use it in your main component (App.js).
- It rendered fine In the browser

### CODE REVIEW

File: src/components/calculate.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```
Comment (Olamarx) [inline src/components/calculate.js]: - [ ] The requirement states that this file name should be created as `Calculator.js`, not `calculate.js`. There is a need for us to make the project requirements to be the first thing, then our own can be secondary.

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @lawrahkonwea, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Good feature branch name ✔️
No linter errors ✔️
Good code organization and working app ✔️
Descriptive README ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -205,21 +124,14 @@ Example:
...
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
```
Comment (HAMMAS-SALEEM) [inline README.md:137]: **[Optional]**

- [ ] You have nothing in the **Future Feature** section of your readme file. Kindly insert data in this section to make your readme file descriptive :blush:

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @lawrahkonwea 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements
✔️ Following Linters

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**